rgit.rb

```diff
@@ -298,7 +303,7 @@ def unbind
 
   desc "test me", "pls"
 
-  def test_me
+  def test_me(notRequired="")
```

Comment:
Unused method argument - `notRequired`. If it's necessary, use `_` or `_notRequired` as an argument name to indicate that it won't be used. You can also write as `test_me(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space missing in default value assignment.